src/components/Game/DeadScreen.tsx

```diff
@@ -7,19 +7,13 @@ import React from 'react'
 import { QueryMode } from '@entur/sdk'
 import { StopPlace } from '@/lib/types/types'
 
-type Props = {
-    mode: QueryMode
-    stopPlace: StopPlace
-}
-function DeadScreen({ mode, stopPlace }: Props): JSX.Element {
+function DeadScreen(): JSX.Element {
     window.scrollTo(0, 0) // Scroll to top of the screen
     return (
         <>
             <Heading2>Du døde!</Heading2>
             <Paragraph>
-                {`Det går ingen avganger med ${getModeTranslation(
-                    mode,
-                ).toLowerCase()} fra ${stopPlace.name} i nær fremtid.`}
+                {`Du brukte desverre for lang tid.`}
```

Comment:
Dessverre skrives med to s-er tror jeg!
Skal det evt stå noe mer engasjerende her, har satt Tomas på saken

Reply:
Takk! Er visst ikke så god på norsk jeg😅